code/API_definitions/qod-api.yaml

```diff
@@ -334,7 +334,14 @@ paths:
         - QoS Sessions
       summary: "Extend the duration of an active session"
       description: |
-        Extend the duration of an active QoS session. If this operation is executed successfully, the new duration of the target session will be the original duration plus the additional duration. The remaining duration plus the requested additional duration shall not exceed the maximum limit. Otherwise, the new remaining duration will be the maximum limit.
+        Extend the duration of an active QoS session. If this operation is executed successfully, the new duration of the target session will be the original duration plus the additional duration.
+        The new remaining duration shall not exceed the maximum duration limit. Otherwise, the new remaining duration will be set to the maximum limit. The remaining duration is calculated as the 'expiresAt' minus the current time of the extendDuration API call.
+        An example: A QoD session was orginally created with duration 80,000 seconds. 10,000 seconds later, the deveoper wants to extend the session with 20,000 seconds.
```

Comment:
```suggestion
        An example: A QoS session was originally created with duration 80,000 seconds. 10,000 seconds later, the developer wants to extend the session with 20,000 seconds.
```

Reply:
The term "maximum duration limit" is ambiguous, because we have two separate limits:
- The maximum **initial** duration when the session is created
- The maximum **remaining** duration if the session is extended

Currently, both limits are set to 86,400 seconds, We may intend that they both always have the same value, but they do not have to have the same value, and the documentation should clarify that they are logically different parameters.

I'll suggest some alternative text to clarify which is intended here.

Reply:
> I'll suggest some alternative text to clarify which is intended here.

@eric-murray Which text you are suggesting?

Reply:
I suggested some alternative text below.

Reply:
@eric-murray thanks a lot, have committed your suggestion.